questions 1-14

```diff
@@ -0,0 +1,96 @@
+1. models.py contains classes and fields where tables represent tables and fields represents the column types.
+   Some field options are null,  blank, default, editable unique and more.
+   models.py is mostly incharge of containing the DBs build state.
+
+2. Views is incharge of the logic in the backend. That's where all of the functionality is done for the clients requests.
+   The functions in the view can retrieve data from the db, send an email and do lots of more backend related logic.
+
+3. The url.py file holds all of the URLs that our project works with.
+   When a client enters a url it checks the url.py and if it exists it then will send the request to a view function
+   that will handle the clients request.
+
+4. Migration is the process of applying changes to your model (db schema). Its incharge of keeping your model and db in sync.
+   To add a new migration you need to write into the cmd line: manage.py migrate.
```

Comment:
Very true.
Also worth noting that to create migrations in the first place you need to run `python manage.py makemigrations`.